docs/walrus-sites/tutorial-install.md

```diff
@@ -44,3 +60,52 @@ Commands:
 
   ⋮
 ```
+
+## Configuration
+
+The `site-builder` tool needs a configuration file to work.
+This file is called `sites-config.yaml` and looks like this:
+
+```yaml
+# module: site
+# portal: walrus.site
+package: 0xc5bebae319fc9d2a9dc858b7484cdbd6ef219decf4662dc81a11dc69bb7a5fa7
+# general:
+#   rpc_url: https://fullnode.testnet.sui.io:443
+#   wallet: /path/to/.sui/sui_config/client.yaml
+#   walrus_binary: /path/to/walrus
+#   walrus_config: /path/to/devnet_deployment/client_config.yaml
+#   gas_budget: 500000000
+```
+
+As you can see, the configuration file is quite simple.
+The only mandatory field is the `package` field,
+which represents the Sui object ID of the Walrus Sites
+smart contract. You can find the latest version of the
+package in the [Walrus Sites repository](https://github.com/MystenLabs/walrus-sites/tree/testnet)
+on the `testnet` branch.
```

Comment:
Suggestion: provide the full command to clone from testnet into one of the default configuration folders

Reply:
Already done! (see line 116)